### PR TITLE
[Debug] Rework a bit the PHP doc

### PR DESCRIPTION
**Pull request by lyrixx**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
:+1:

File: src/Symfony/Component/Debug/ErrorHandler.php [hunk 1]
```diff
@@ -282,7 +282,7 @@ public function scopeAt($levels, $replace = false)
     }
 
     /**
-     * Sets the error levels that are logged with their stack trace.
+     * Sets the PHP error levels for which stack trace are preserved.
```
Comment (javiereguiluz) [inline src/Symfony/Component/Debug/ErrorHandler.php]: Typo. Replace it by either `... stack trace is preserved.` or `... stack traces are preserved.`

Comment (lyrixx) [inline src/Symfony/Component/Debug/ErrorHandler.php]: Good catch. Thanks.

**Comment by nicolas-grekas** [comment]
Thank you @lyrixx.

**Pull request closed without merging**